docs/developing.md

```diff
@@ -4,41 +4,45 @@
 
 On all platforms, the minimum requirements are:
 
-- Git client and command line tools
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
 - [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [.NET 7.0 SDK](https://dotnet.microsoft.com/download/dotnet/7.0)
 
 ### Windows
 
-- [Visual Studio 2022 (17.1)](https://visualstudio.microsoft.com/downloads/) or higher
+- [Visual Studio 2022 (17.4)](https://visualstudio.microsoft.com/downloads/)
+  or higher
   - Workloads with the following components:
     - ASP.NET and web development
     - .NET desktop development
     - Desktop development with C++
   - Individual components:
     - MSVC v142 - Visual Studio 2019 C++ x64/x86 build tools (version 14.29)
-- [Docker for Windows](https://docs.docker.com/docker-for-windows/)
 
 Microsoft provides
 [evaluation developer VMs](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines)
 with Windows and Visual Studio preinstalled.
 
-### Linux and macOS
+### macOS
 
-- cmake, make, gcc, clang, and clang++
-- [Docker](https://docs.docker.com/engine/install/)
-- [Docker Compose](https://docs.docker.com/compose/install/)
+- Run: `xcode-select --install`
+- [Visual Studio 2022 for Mac (17.4)](https://visualstudio.microsoft.com/vs/mac/)
```

Comment:
Is this really required? Or is it here because brings dependencies in one go?

Reply:
> Or is it here because brings dependencies in one go?

I will remove it 😉

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1720/commits/ea749237f709cda9ca90155fb4e6563ab371c308